nhais/sequence/tests/component_test_transaction_id.py

```diff
@@ -0,0 +1,78 @@
+"""Module to test transaction id generating"""
+import unittest
+import aioboto3
+from sequence import dynamo_sequence
+from sequence.transaction_id import TransactionIdGenerator
+from utilities import test_utilities
+from utilities import config
+
+
+class ComponentTestTransactionId(unittest.TestCase):
+
+    def setUp(self):
+        config.setup_config("MHS")
+        self.table_name = 'test_dynamo_sequence_table'
+        self.key = 'transaction_id'
+        self.endpoint = config.get_config('DYNAMODB_ENDPOINT_URL', None)
+        self.region_name = 'eu-west-2'
+        self.transaction_id_generator = TransactionIdGenerator()
+        # for test purpose we use another table so we need to override the table name in the generator
+        self.transaction_id_generator.table_name = self.table_name
+
+    @test_utilities.async_test
```

Comment:
These component tests should all be in component_test_dynamo_sequence. They don't actually test anything implemented by TransactionIdGenerator.

If you were to test the TransactionIdGenerator class it should be unit tested with the DynamoSequenceGenerator mocked out. See usages of "@mock" annotations for examples.

Reply:
moved, this mock will be added later